debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/SqlServerConnectorIT.java

```diff
@@ -1290,6 +1290,85 @@ public void excludeColumnWhenCaptureInstanceExcludesColumns() throws Exception {
         stopConnector();
     }
 
+    @Test
+    @FixFor("DBZ-2522")
+    public void excludeColumnWhenCaptureInstanceExcludesColumnInMiddleOfTable() throws Exception {
+        connection.execute(
+                "CREATE TABLE blacklist_column_table_a (id int, amount integer, name varchar(30), primary key(id))");
```

Comment:
We're [moving away](https://debezium.io/blog/2020/09/03/debezium-1-3-beta1-released/) from the terms blacklist and whitelist. Could you please use another name, e.g. "table_with_excluded_column"?

Reply:
Yeah I can make that change! I was copying from an older branch and thought I changed all the occurrences, I'll make sure I didn't miss anymore

Reply:
Thanks!